Assets/Resources/Prefabs/Popups/DamagePopup.prefab

```diff
@@ -185,3 +185,4 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: c692c9b45b1cb4795a87ae10289a110e, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+  destroyTimeSeconds: 0.5
```

Comment:
i added a variable for how long we want the popups to linger after they are instantiated. i find a full second is a little long for my taste (also means we need more objects in the pooler, since it takes longer to recycle a single one)